test/recipes/controls/aws_parallelcluster_config/networking_spec.rb

```diff
@@ -0,0 +1,24 @@
+# Copyright:: 2023 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License").
+# You may not use this file except in compliance with the License. A copy of the License is located at
+#
+# http://aws.amazon.com/apache2.0/
+#
+# or in the "LICENSE.txt" file accompanying this file.
+# This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, express or implied.
+# See the License for the specific language governing permissions and limitations under the License.
+
+control 'networking_configured' do
+  title 'Check that networking has been configured'
+
+  only_if { !os_properties.virtualized? }
```

Comment:
You can remove this check, we already have this kind of tests for example for sticky bits

Reply:
Are you sure? From what I can see, in the [sticky_bits_spec](https://github.com/aws/aws-parallelcluster-cookbook/blob/redhat8/test/resources/controls/aws_parallelcluster_config/sticky_bits_spec.rb#L4) we use `!os_properties.virtualized?`